tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -43,11 +46,16 @@ class PrecisionParametrizedTest
  protected:
   // Allocation order set by the pass breaks matmul tests
   // see issue https://github.com/NVIDIA/Fuser/issues/1810
-  PrecisionParametrizedTest() : optimization_guard_(false) {}
+  PrecisionParametrizedTest() : optimization_guard_(false) {
+    DisableOptionsGuard::getCurOptions().set(DisableOption::MatmulExprEval);
+  }
 
  private:
   preseg_passes::OptimizationPassGuard<preseg_passes::AllocationDomainPass>
       optimization_guard_;
+
+ private:
+  DisableOptionsGuard opt_guard_;
```

Comment:
```suggestion
  DisableOptionsGuard option_guard_;
```

There's already a `private` before this. Also, I'd rename this to option_guard_ to distinguish it from optimization_guard_.